src/util/introduction-ctoj-neg06.js

```diff
@@ -116,11 +120,11 @@ export function addIntroduction(timeline, options) {
         trial.data.refreshRate = Math.round(rate);
       },
       on_finish: (trial) => {
-        const responses = JSON.parse(trial.responses);
+        const responses = trial.response;
         const queryString = window.location.search;
         const urlParams = new URLSearchParams(queryString);
-        let participant_code = jatos.urlQueryParameters.PROLIFIC_PID;
-        if (participant_code === null) {
+        let participant_code = urlParams.get("PROLIFIC_PID");
+        if (participant_code === undefined) {
```

Comment:
`urlParams.get` will return `null` if the search parameter is not found. See [URLSearchParams docs](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get) for reference.

Therefore `participant_code` should be checked for `null` instead of `undefined`.
Otherwise there will be an error at line 133: `participantCode: md5(participant_code)` if `IS_A_PROLIFIC_STUDY` is set to `true` in `color-toj-negation-06.js` and no parameter is given in the url because `participant_code` is not set in this case since `null === undefined` is `false`. 

```suggestion
        if (participant_code === null) {
```

Reply:
Thanks!